### DISCUSSION TITLE
Defining ffmpeg and ffprobe paths manually results in a 500 HttpException

### DISCUSSION THREAD

**Discussion by flossgeek** [LycheeOrg/Lychee#2488] [Q&A] [answered]
I tried to set the ffmpeg_path and ffmprobe_path manually via the Settings page, but this results in a 500 HttpException which says: `Given path () to the ffprobe binary is invalid`.

I've tried a number of different ways to define the location of the binaries, but I assume it should just be an absolute path? The binaries themselves are executable (I put them in a bin folder within the public folder and gave them the correct permissions).

Populating the paths manually in the php files also does not work anymore (this partly worked in the past), but I prefer doing it via the settings if that's at all possible.

Thanks for the support!

**Comment by ildyria** [comment]
Investigating it.

**Reply by ildyria** [reply]
PR on dependency: https://github.com/LycheeOrg/php-exif/pull/73
Then I should be able to hopefully fix the code in Lychee.

**Reply by ildyria** [reply]
@flossgeek could you try [fix-phpexif](https://github.com/LycheeOrg/Lychee/tree/fix-phpexif) ?

**Reply by flossgeek** [reply]
Thanks for responding so fast!
I replaced the `php-exif` folder in `vendor/lychee-org` with the new release, but this doesn't help on my side. I'm still getting the same error message (ffprobe is invalid for videos and exiftool is invalid for photos).

**Reply by ildyria** [reply, marked as answer]
I updated master so using master and composer install should put you with the latest.

**Reply by ildyria** [reply]
> I assume it should just be an absolute path

yes.

**Reply by ildyria** [reply]
@flossgeek please try current master. :)

**Reply by flossgeek** [reply]
Ah, yes, I'm sorry. It works well now, thanks for fixing it!
